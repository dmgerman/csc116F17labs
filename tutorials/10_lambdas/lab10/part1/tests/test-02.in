100 200 600 900 106 116 111 -1
Monday Tuesday Wednesday Thursday Friday