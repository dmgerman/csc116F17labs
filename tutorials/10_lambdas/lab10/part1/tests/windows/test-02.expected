Enter a sequence of integers, ending with -1: 
Vector VI: 100 200 600 900 106 116 111 
Enter a sequence of words: 
Vector VS: Monday Tuesday Wednesday Thursday Friday 

print_vector_alternating: 100 600 106 111 

double_vector_elements: MondayMonday TuesdayTuesday WednesdayWednesday ThursdayThursday FridayFriday 

negate_alternating: -100 200 -600 900 -106 116 -111 

print_vector_offset (offset 6): 106 206 606 906 112 122 117 

print_vector_offset (offset 9): 109 209 609 909 115 125 120 

sum_vector(VI): 2133
sum_vector(VS): MondayTuesdayWednesdayThursdayFriday

sum_vector_templated(VI): 2133
sum_vector_templated(VS): MondayTuesdayWednesdayThursdayFriday

longest_string(VS): Wednesday

